public/index.css

```
@@ -0,0 +1,99 @@
+*{
+    margin:0;
+    padding:0;
+    box-sizing:border-box;
+}
+.app{
+    display:grid;
+    height: 100vh;
+    overflow: hidden;
+    grid-template-columns:300px 1fr 300px;
+    grid-template-rows: min-content  1fr min-content;
+    font-family: 'Segoe UI',Tahoma,'Segoe UI', Tahoma, Geneva, Verdana, sans-serif;
+}
+
+.header2{
+    display: flex;
+    align-items: center;
+    padding: 4px 310px;
+    background-color: black;
+    height: 70px;
+    grid-column: 1/4;
+    font-size: 18px;
+}
+.socset{
+    align-items: center;
+    width: 170 px;
+}
+.title{
+color: white;
+font-size: 25px;
+}
+    .title2{
+    padding:4px 30px 4px 550px;
+    color: white;
+    font-size: 15px;
+    }
+    .title3{
+        padding: 30px;
+        color: white;
+        font-size: 15px;
+    }
+    .title5{
+        color: white;
+        font-size: 15px;
+    }
+.accordion1{
+    background-color:  rgb(37, 57, 80);
+}
+.accordion2{
+    display:grid;
+    grid-template-columns: 2fr 7fr;
+    grid-template-rows: min-content   1fr ;
+    background-color:white ;
+}
+.dop1{
+    background-color:blue ;
+    height: 200px;
+    grid-column: 1/3;
+}
+.titledop{
+    padding: 35px;;
+    color: white;
+    font-size: 35px;
+}
+.dop2{
+    background-color:rgb(19, 19, 24) ;
+}
+.title2dop1{
+    padding: 10px;
+    color: white;
+    font-size: 18px;
+}
+.dop3{
+    background-color:white ;
+}
+.title3dop1{
+    padding: 50px 30px 30px 50px ;
+    color: black;
+    font-size: 40px;
+}
+.title3dop2{
+    padding: 0px 30px 30px 50px ;
```

```suggestion
    padding: 0px 30px 30px 50px ;
```
вот это тоже можно выделить , раз отступы одинаковые